dipy/data/fetcher.py

```diff
@@ -591,6 +597,125 @@ def read_syn_data():
 """
 
 
+def fetch_viz_icons():
+    """ Download icons for visualization
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
```

Comment:
Okay sure. But can you tell me why Dropbox is bad? I think we never had any problem with Dropbox. And I suppose if we move all the Dropbox files to NITRC we should do that all in one PR and not here. What do you think?